core/src/main/java/org/apache/accumulo/core/fate/user/schema/FateSchema.java

```diff
@@ -65,4 +60,15 @@ public static class RepoColumnFamily {
     public static final Text NAME = new Text(STR_NAME);
   }
 
+  public static class TxAdminColumnFamily {
```

Comment:
Could add a comment like, "when FATE looks for work, this is the family scanned"